packages/gp2-components/src/organisms/__tests__/BiographyModal.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { gp2 as gp2Fixtures } from '@asap-hub/fixtures';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ComponentProps } from 'react';
+import { StaticRouter } from 'react-router-dom';
+import BiographyModal from '../BiographyModal';
+
+describe('BiographyModal', () => {
+  const getSaveButton = () => screen.getByRole('button', { name: 'Save' });
+
+  beforeEach(jest.resetAllMocks);
+  type BiographyModalProps = ComponentProps<typeof BiographyModal>;
+  const defaultProps: BiographyModalProps = {
+    ...gp2Fixtures.createUserResponse(),
+    backHref: '',
+    onSave: jest.fn(),
+  };
+
+  const renderModal = (overrides: Partial<BiographyModalProps> = {}) =>
+    render(
+      <StaticRouter>
+        <BiographyModal {...defaultProps} {...overrides} />
+      </StaticRouter>,
+    );
+
+  it('renders a dialog with the right title', () => {
+    renderModal();
+    expect(screen.getByRole('dialog')).toContainElement(
+      screen.getByRole('heading', { name: 'Biography' }),
+    );
+  });
+
+  it('renders biography', () => {
+    renderModal();
+    expect(
+      screen.getByRole('textbox', {
+        name: 'Background (Required)',
```

Comment:
will a regex for background only work here?

Reply:
Done